### ISSUE TITLE
Editor can't start because of dxgi.dll

### ISSUE THREAD

**Issue by Tamren** [EnderHDMC/MHWISaveEditor#6]
I'm getting an error message when I try to start the editor:

"error on startup: "the procedure entry point createdxgifactory2 could not be located in the dynamic link library dxgi.dll"

I'm trying to use v1.4, older versions from v1.1 to v1.3 of the save editor work normally.

**Comment by EnderHDMC** [comment]
What version of Windows are you on?

**Comment by Tamren** [comment]
Oh that didn't occur to me, I'm running MHW on an old windows 7 pc I keep around for legacy games. Is that the issue?

**Comment by EnderHDMC** [comment]
It seems as though that issue only occurs on Windows 7 since it doesn't have DirectX 12.
That being said, this shouldn't be an issue in the first place, the editor doesn't use DirectX.

**Comment by EnderHDMC** [comment]
Looking into it quickly it seems as though Qt 6.2.2 drops official support for Windows 7.
In commit f5a9ccdd5cd09cd84b456219b5bb18fb84aa55a1, I had updated the project to use 6.2.3.

This can be reverted to fix the issue, as I only updated because I wanted to clear my storage of things I wasn't using.

Looking at the imports of Qt6Gui.dll using dumpbin it can be seen that it imports createdxgifactory2 from dxgi.dll.
After reverting to 6.1.3, it can then be seen that createdxgifactory2 is no longer imported by Qt6Gui.dll.
